src/mjolnir/osmway.cc

```diff
@@ -182,6 +182,13 @@ std::vector<std::string> OSMWay::GetTaggedNames(const UniqueNames& name_offset_m
       names.emplace_back(std::to_string(static_cast<uint8_t>(TaggedName::kTunnel)) + t);
     }
   }
+  if (name_pronunciation_index_ != 0) {
+    // name:pronunciation names
+    tokens = GetTagTokens(name_offset_map.name(name_pronunciation_index_));
+    for (const auto& t : tokens) {
+      names.emplace_back(std::to_string(static_cast<uint8_t>(TaggedName::kPronunciation)) + t);
```

Comment:
Does Valhalla still use tags like `alt_name`, `official_name`, and `name:en` in guidance instructions, as claimed in #2254? If so, [other `:pronunciation` tags](https://taginfo.openstreetmap.org/search?q=pronunciation) may be relevant, though `name:pronunciation` is the most prominent of them.

Reply:
`alt_name` is a conditional config parameter when building tiles now so you can turn it off. `official_name` is still added as a name regardless of configuration in addition to `name` and also `name:en`. you can check this function for more details: https://github.com/valhalla/valhalla/blob/6add48fa3e283c159ab03f1bc5ed5b4722acc7a2/src/mjolnir/osmway.cc#L99

in the future we'll be able to store language specific tagging (now that we have tagged values support) so we should be able to "undo" the `name:en` fallback to be a bit more graceful